cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,38 @@
-/// <reference types='cypress' />
+/// <reference types="cypress" />
 
-describe('Book Store app', () => {
+describe('Book Store App', () => {
   before(() => {
-    
+    cy.viewport(1920, 1080);
+    cy.visit('https://demoqa.com/login');
   });
 
-  it('', () => {
-    
+  beforeEach(() => {
+    cy.reload();
+  });
+
+  it('Login', () => {
```

Comment:
the name of the test should say what it does and it should create a sentence together with the name of the describe

Reply:
Done